package/gluon-config-mode-mesh-vpn/luasrc/lib/gluon/config-mode/reboot/0100-mesh-vpn.lua

```diff
@@ -1,33 +1,34 @@
 local uci = luci.model.uci.cursor()
 local meshvpn_enabled = uci:get("fastd", "mesh_vpn", "enabled", "0")
 
-if meshvpn_enabled ~= "1" then
-  return nil
-else
-  local i18n = require "luci.i18n"
-  local util = require "luci.util"
+local i18n = require "luci.i18n"
+local util = require "luci.util"
 
-  local gluon_luci = require 'gluon.luci'
-  local site = require 'gluon.site_config'
-  local sysconfig = require 'gluon.sysconfig'
+local gluon_luci = require 'gluon.luci'
+local site = require 'gluon.site_config'
+local sysconfig = require 'gluon.sysconfig'
 
-  local pretty_hostname = require 'pretty_hostname'
+local pretty_hostname = require 'pretty_hostname'
 
-  local pubkey = util.trim(util.exec("/etc/init.d/fastd show_key " .. "mesh_vpn"))
-  local hostname = pretty_hostname.get(uci)
-  local contact = uci:get_first("gluon-node-info", "owner", "contact")
+local pubkey = ""
+local hostname = pretty_hostname.get(uci)
+local contact = uci:get_first("gluon-node-info", "owner", "contact")
 
+if meshvpn_enabled ~= "1" then
+  local msg = i18n.translate('gluon-config-mode:novpn')
+else
+  pubkey = util.trim(util.exec("/etc/init.d/fastd show_key " .. "mesh_vpn"))
   local msg = i18n.translate('gluon-config-mode:pubkey')
```

Comment:
You changed the whole control flow of the function, the use site of *msg* (the return clause at the end) was inside the same if branch before. You moved the return out of the if/else, so *msg* is now out of scope there!